src/components/icons/integrations.js

```diff
@@ -0,0 +1,292 @@
+import React from "react";
+
+export const JekyllIcon = props => (
+  <svg
```

Comment:
Use our `Svg` component instead 🙂

Like this https://github.com/microlinkhq/www/blob/master/src/components/icons/External.js#L6